docs/network-flow-visibility.md

```diff
@@ -283,6 +285,9 @@ flow-aggregator.conf: |
   # Provide DNS name or IP address of flow aggregator for generating TLS certificate. It must match
   # the flowCollectorAddr parameter in the antrea-agent config.    
   #flowAggregatorAddress: "flow-aggregator.flow-aggregator.svc"
+
+  # Provide record format to send to the configured flow collector, which is IPFIX or JSON.
```

Comment:
nit: this doesn't match the actual configuration files

Reply:
Done. Thanks!